### DISCUSSION TITLE
Setup error

### DISCUSSION THREAD

**Discussion by MarcosGraficon** [OCA/odoo-community.org#137] [Q&A] [answered]
Hey, i just downloaded and isntalled fieldservice and base_territory, but no groups were created and no element menu either, im missing something but dont know what , im working in Spansih dont know if that affects 

![image](https://github.com/OCA/odoo-community.org/assets/156304478/1552e22b-ff3a-40e4-9615-aff95eaf453a)

![image](https://github.com/OCA/odoo-community.org/assets/156304478/70018e43-f839-4186-a939-7e93b1cfe33c)

**Comment by chienandalu** [comment, marked as answer]
The group names will be translated. It will be something like "Servicio de campo". You can spot the precise term in the `es.po` translation files. Otherwise, you can just switch to english and do the setup in that language context.
